packages/reqresp/package.json

```diff
@@ -69,7 +69,7 @@
     "@lodestar/types": "^1.8.0"
   },
   "peerDependencies": {
-    "libp2p": "0.41.0"
+    "libp2p": "~0.42.2"
```

Comment:
This fixes the warning if beacon-node package is installed, however top-level install still shows a warning, not sure how to get rid of that, might be related to
-  https://github.com/yarnpkg/yarn/issues/4433

Checked peer deps via [check-peer-dependencies](https://yarnpkg.com/package/check-peer-dependencies), looks mostly correct now